app/models/admin/FeatureFlags.scala

```diff
@@ -93,23 +96,16 @@ case object TaxcalcMakePaymentLinkToggle extends FeatureFlagName {
   override val description: Option[String] = Some("Enable/disable direct link to make a payment on taxcalc tile")
 }
 
-case object NpsShutteringToggle extends FeatureFlagName {
-  override val name: String                = "nps-shuttering-toggle"
-  override val description: Option[String] = Some("Enable/disable the nps shuttering banner and subpage")
+case object ShowNpsOutageBannerToggle extends FeatureFlagName {
```

Comment:
rename ShowNpsOutageBannerToggle to ShowOutageBannerToggle

Reply:
Done